warehouse/templates/manage/account.html

```diff
@@ -540,7 +540,7 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
       {% elif event.tag == EventTag.Account.LoginSuccess %}
       <strong>{% trans %}Logged in{% endtrans %}</strong><br>
       <small>
-        {% trans %}Two factor method:{% endtrans %}
+        {% trans %}Two factor method:{% endtrans %} 
```

Comment:
This is intentional to pad the `:` and make the event display a little prettier.